### ISSUE TITLE
provide screenreader hints on first workspace focus

### ISSUE THREAD

**Issue by maribethb** [RaspberryPiFoundation/blockly#9621]
### Check for duplicates

- [ ] I have searched for similar issues before opening a new one.

### Description

The first time a user focuses the workspace, the aria live region should announce some help that tells screenreader users how to navigate.

@microbit-robert do you know what the makecode helps offers? should we use the same text? not sure how much text you all used in the user testing or which things are most important to announce.

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

**Comment by maribethb** [comment]
the hint should tell you to use the shortcut from #9861 to enable screenreader optimizations
